core/src/main/java/nl/nn/adapterframework/processors/CorePipeLineProcessor.java

```diff
@@ -193,16 +158,9 @@ public PipeLineResult processPipeLine(PipeLine pipeLine, String messageId, Messa
 							log.debug("wrapping PipeLineResult");
 							PipeRunResult wrapResult = pipeProcessor.processPipe(pipeLine, outputWrapper, message, pipeLineSession);
 							if (wrapResult!=null && !wrapResult.getPipeForward().getName().equals("success")) {
-								PipeForward wrapForward=wrapResult.getPipeForward();
-								if (wrapForward.getPath()==null) {
-									throw new PipeRunException(pipeToRun,"forward ["+wrapForward.getName()+"] of outputWrapper has emtpy forward path");
-								}
-								log.warn("setting next pipe to ["+wrapForward.getPath()+"] due to wrap fault");
+								forwardTarget = pipeLine.resolveForward(outputWrapper, wrapResult.getPipeForward());
+								log.warn("setting next pipe to ["+forwardTarget.getName()+"] due to wrap fault");
```

Comment:
hier ook weer, dit is toch runtime niet configure time? is het dan niet 'missing forward for [xxx] forwarding to pipe [xxx] due to wrap fault?